Website/src/items/rendering-contents-resolvers/Edge Rendering Contents Resolvers/Context Item Children Resolver Excluding All But News.yml

```diff
@@ -0,0 +1,41 @@
+﻿---
+ID: "91963359-b9e8-4f9d-b08e-2672fb222def"
+Parent: "33b97084-9b0b-4f01-9b4a-531e70b72c7a"
+Template: "814c1dbb-b118-4511-ba2b-7288ed3d490b"
+Path: /sitecore/system/Modules/Layout Service/Rendering Contents Resolvers/Edge Rendering Contents Resolvers/Context Item Children Resolver Excluding All But News
```

Comment:
Please re-serialize this item using `serpull.ps1`. Its indentation is different from other serialized items. It has too many spaces. This might be due to a manual open/save in your test editor and an enabled  "format on save" option.